Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using AzureCognitiveSearch.PowerSkills.Common;
+using Newtonsoft.Json.Linq;
+
+// languages used for Azure Search with Text Analytics:
+// el, th, he, tr, cs, hu, ar, ja-jp, fi, da, no, ko, pl, ru, sv, ja, it, pt, fr, es, nl, de, en
+// greek, thai, hebrew, turkish, czech, hungarian, arabic, japanese, finnish, danish, norwegian, korean, polish, russian, swedish, japanese (again??), 
+// italian, portuguese, french, spanish, dutch, german, english
+// unicode blocks in order:
+// InThai, InHebrew
+
+namespace AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch
+{
+    /// <summary>
+    /// Based on sample custom skill provided in Azure Search. Provided a user-defined list of entities
+    /// this function determines the Entity first occurrence within a given document. This list of entities
+    /// must repeatedly be provided by the user for each document.
+    /// </summary>
+    public static class CustomEntitySearch
+    {
+        private static int MaxRegexEvalTime = 1;
+        /// <summary>
+        /// We assert the following assumptions:
+        /// 1. All text files are Latin based (i.e. no languageCode)
+        /// 2. Words can contain special characters and numbers
```

Comment:
this still true?

Reply:
No, sorry. The files do not only need to be latin based for exact match based on a new assumption that the files are 64 encoded